### PR TITLE
add code of conduct

### PR DESCRIPTION
**Pull request by AbednegoTM**
This is a draft PR for #6 with regards code of conduct. I opened it as a draft  PR because it may require discussing adoption and needs contact information for people responsible for code of conduct enforcement. 
If what is contained in the CoC is applicable to the project I can go ahead and mark the PR ready for review so that it can be reviewed and merged.

### CODE REVIEW

**Comment by getify** [comment]
I think this looks really good. I would definitely list my name and email as one of the community leaders for the purposes of enforcement:

Kyle Simpson getify@gmail.com

But I also think we should ask for at least 2-3 other people willing to do so, publicly and on the record. And I think it would be great for that to include a variety of people of different groups (women, persons of color, etc) so we make sure there's fair and equal governance of our mutual conduct.

If you are interested in joining the group of leaders as it relates to the code-of-conduct here, please speak up! :)

**Comment by AbednegoTM** [comment]
Okay cool sounds great , I can add myself and add two more people by opening an issue calling for enforcers. Those interested can use this format to describe themselves :-

Name :
Email :
Nationality :
Race : 
Gender Identity :

Then you can pick two from the list. How does that sound ?.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at : 
+
+ - Kyle Simpson getify@gmail.com
```
Comment (pselle) [inline CODE_OF_CONDUCT.md:65]:
```suggestion
 - Kyle Simpson getify@gmail.com
 - Pam Selle pam@thewebivore.com
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+ - Pam Selle pam@thewebivore.com
```
Comment (getify) [inline CODE_OF_CONDUCT.md:66]:
```suggestion
 - Pam Selle pam@thewebivore.com
 - Aditi Ranka aditi.ranka@gmail.com
 - [TODO: full name] artnerdnet@gmail.com
 - Sandra Molina molina.sandra.101@gmail.com
 - Nima Mohamadian [TODO: contact info]
 - Abednego Mwanza [TODO: contact info]
```

Comment (getify) [inline CODE_OF_CONDUCT.md:66]:
```suggestion
 - Pam Selle (@pselle) pam@thewebivore.com
 - Aditi Ranka (@aditiranka27) aditi.ranka@gmail.com
 - @artnerdnet artnerdnet@gmail.com
 - Sandra Molina (@sandramolina) molina.sandra.101@gmail.com
 - Nima Mohamadian (@novonimo) nimamohamadian89@gmail.com
 - Abednego Mwanza (@AbednegoTM) abednegomwanza@gmail.com
 - Prabhpreet Singh (@prabhpreet-code) prabhpreetsingh.singh0203@gmail.com
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at : 
+
+ - Kyle Simpson getify@gmail.com
```
Comment (getify) [inline CODE_OF_CONDUCT.md:65]:
```suggestion
 - Kyle Simpson (@getify) getify@gmail.com
```

**Review by getify — APPROVED** [review]

**Comment by getify** [comment]
This looks good for now. We can always revisit if we need to.

**Pull request merged**